packages/loader/container-loader/src/container.ts

```diff
@@ -398,20 +396,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
     private _attachState = AttachState.Detached;
 
-    private readonly _storage: ContainerStorageAdapter;
+    private readonly storageService: ContainerStorageAdapter;
     public get storage(): IDocumentStorageService {
-        return this._storage;
+        return this.storageService;
     }
 
-    private _storageService: IDocumentStorageService & IDisposable | undefined;
-    private get storageService(): IDocumentStorageService {
-        if (this._storageService === undefined) {
-            throw new Error("Attempted to access storageService before it was defined");
-        }
-        return this._storageService;
-    }
-
-    private readonly clientDetailsOverride: IClientDetails | undefined;
+    readonly clientDetailsOverride: IClientDetails | undefined;
```

Comment:
Is this public now?

Reply:
huh. didn't mean to change that
```suggestion
    private readonly clientDetailsOverride: IClientDetails | undefined;
```